.github/workflows/nix.yaml

```
@@ -112,19 +111,54 @@ jobs:
         run: |
           om ci run --systems "${{ matrix.system }}" --no-out-link -- -v
 
-      - name: Build Docker image
+      - name: Build And Push Docker image
         id: docker
-        if: steps.check-conditions.outputs.should_build == 'true' && matrix.system == 'x86_64-linux' && needs.check-conditions.outputs.should-push-docker == 'true'
+        if: steps.check-conditions.outputs.should_build == 'true' && (matrix.system == 'x86_64-linux' || matrix.system == 'aarch64-linux') && needs.check-conditions.outputs.should-push-docker == 'true'
+        env:
+          DOCKER_REGISTRY: ghcr.io
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:36: Double quote to prevent globbing and word splitting

(shellcheck)